packages/front-end/components/Features/ConditionInput.tsx

```diff
@@ -223,10 +223,12 @@ export default function ConditionInput(props: Props) {
                   <div className="col-sm-12 col-md mb-2">
                     <SelectField
                       value={field}
-                      options={attributeSchema.map((s) => ({
-                        label: s.property,
-                        value: s.property,
-                      }))}
+                      options={attributeSchema
+                        .filter((s) => !s.archived)
```

Comment:
There are a bunch of other places in the code where we need to add this filter logic (I counted 14).  They all use the `useAttributeSchema` hook, so maybe we can just add the filter there instead.

The two places that DO need the full list of attributes are the attributes page and modal.  For those, we can get the attributeSchema another way (e.g. the `useOrgSettings` hook).